google/cloud/bigtable/quickstart/quickstart.cc

```diff
@@ -57,3 +59,5 @@ int main(int argc, char* argv[]) try {
   std::cerr << "Standard C++ exception raised: " << ex.what() << "\n";
   return 1;
 }
+
```

Comment:
Could we remove these blank lines (which, it seems, would eliminate the `README.md` edit)?